pocketoptionapi_async/websocket_client.py

```
@@ -211,85 +211,64 @@ async def connect(self, urls: List[str], ssid: str) -> bool:
+
+            finalData.tradable_assets = {
+                k: v for k, v in parsed_assets.items() if v["tradable"]
+            }
+
+            await self._emit_event("payout_update", finalData)
 
-        except json.JSONDecodeError as e:
-            logger.error(
-                f"Failed to decode JSON from payout message '{message[:100]}...': {e}"
-            )
         except Exception as e:
```

Ruff (0.15.2)

[warning] Do not catch blind exception: `Exception`

(BLE001)